contents/newsletter/first-time-founders.md

```diff
@@ -0,0 +1,134 @@
+---
+title: How first-time founders fail
+date: 2024-04-10
+author:
+  - andy-vandervell
+featuredImage: >-
+  https://res.cloudinary.com/dmukukwp6/image/upload/posthog.com/contents/images/newsletter/beyond-10x-engineer/super-hog.png
+featuredImageType: full
+tags:
+  - Being a founder
+  - Product-market fit
+crosspost:
+  - Founders
+  - Blog
+---
+
+Read the footnote if "trust me bro" isn't enough for you, but statistically:
```

Comment:
Footnote doesn't exist yet. Just imagine it.

Reply:
Still kinda feel this falls flat as an opening line. Telling people to read a footnote in the first sentence is weird? You could turn the bullets into a par, so like:

Which kind of founder is most likely to fail when launching a new startup? Is it a first-time founder? A founder who's previously failed? Or a previously successful founder? Yup, you guessed right, it's the first time founder. This isn't just a vibe, it's backed up by statistics (head to the footnote if you want the numbers.)

But if you want to learn how first-time founders fail, and avoid their mistakes, then you better talk to founder who failed and then found success. We've done it so you don't have to. Here's what they told us.

something like this?

Reply:
Will take another stabs this today